checker.go

```diff
@@ -135,10 +135,10 @@ func (c *cmpEqualsChecker[T]) Check(note func(key string, value any)) (err error
 			err = BadCheckf("%s", r)
 		}
 	}()
-	if diff := cmp.Diff(c.got, c.want, c.opts...); diff != "" {
+	if diff := cmp.Diff(c.want, c.got, c.opts...); diff != "" {
 		// Only output values when the verbose flag is set.
 		note("error", Unquoted("values are not deep equal"))
-		note("diff (-got +want)", Unquoted(diff))
+		note("diff (+got -want)", Unquoted(diff))
```

Comment:
I'd personally make these `-want +got`, leaving the "before" or "minus" file first, because that's what almost everyone would expect.

Reply:
Done